lightning/src/ln/channelmonitor.rs

```diff
@@ -586,6 +586,9 @@ enum OnchainEvent {
 	HTLCUpdate {
 		htlc_update: (HTLCSource, PaymentHash),
 	},
+	MaturingOutput {
```

Comment:
nit: in `fn block_disconnected` the comment needs updating to note that this may be discarded on reorg.